tests/integration-tests/processors/processors-test.js

```diff
@@ -3,11 +3,14 @@ const should = require('should'),
 
 let validHeaders = { 'Content-Type': 'application/json' };
 const requestSender = require('./helpers/requestCreator');
+const testsRequestSender = require('../tests/helpers/requestCreator');
 const { ERROR_MESSAGES } = require('../../../src/common/consts');
+const basicTest = require('../../testExamples/Basic_test.json');
 describe('Processors api', function() {
     this.timeout(5000000);
     before(async function () {
         await requestSender.init();
```

Comment:
maybe rename requestSender to processorRequestSender

Reply:
done